docs/docs/spark-structured-streaming.md

```diff
@@ -20,11 +20,11 @@ title: "Structured Streaming"
 
 # Spark Structured Streaming
 
-Iceberg uses Apache Spark's DataSourceV2 API for data source and catalog implementations. Spark DSv2 is an evolving API with different levels of support in Spark versions.
+Spark Iceberg uses Apache Spark's DataSourceV2 API for data source and catalog implementations. Spark DSv2 is an evolving API with different levels of support in Spark versions.
```

Comment:
```suggestion
Spark Iceberg Connector uses Apache Spark's DataSourceV2 API for data source and catalog implementations. Spark DSv2 is an evolving API with different levels of support in Spark versions.
```

Reply:
Makes sense! I think Iceberg refers to the "iceberg" input format, but that's not totally obvious.